### DISCUSSION TITLE
Absolute Path Navigation

### DISCUSSION THREAD

**Discussion by zambony** [spacedriveapp/spacedrive#2703] [Feature Request]
It would be helpful if there was an address bar to type a local absolute file path into. If I'm browsing local files, sometimes I want to go to the absolute folder of something by typing the path. In Dolphin, I can press CTRL+L and start typing or copy the current path.

**Comment by iLynxcat** [comment]
Agreed this would be a very good feature, checking for duplicates first then making an issue for this if I find none

**Comment by iLynxcat** [comment]
https://github.com/spacedriveapp/spacedrive/issues/2705
